scripts/generateLessTokens.mjs

```diff
@@ -0,0 +1,24 @@
+import { colors } from '../dist/index.js'
```

Comment:
Good idea to write this script and also to not commit the files generated by it!